README.md

```diff
@@ -33,7 +33,8 @@ terminus(server, {
 
   // cleanup options
   timeout: 1000,                   // [optional = 5000] number of milliseconds before forcefull exiting
-  onSigterm,                       // [optional] cleanup function, returning a promise
+  signal,                          // [optional = 'SIGTERM'] what signal to listen for relative to shutdown
```

Comment:
how about renaming to `killSignal`?

Reply:
also, can we keep `onSigterm` too? this way it wouldn't be a breaking change

Reply:
I would argue that it be kept but not publicised. With this change, `onSigterm` could technically be a function relating to any signal a user passes, and doesn't necessarily have to do with SIGTERM. I've coded up the changes, just in the airport and it's not wanting to push. but I've kept it and it will be used if the `onKillSignal` fn doesn't exist

Reply:
great, thanks, looking forward to your changes!